Hyperledger_Fabric_Technical_Charter.md

```diff
@@ -1,116 +1,169 @@
 
-      1. coordinating the technical direction of the Project;
+        1.  **Contributors** include anyone in the technical community that contributes code, documentation, or other technical artifacts to any recognized **Sub-Project** in the Project;
 
-      2. approving project or system proposals (including, but not limited to, archiving and changing a sub-project’s scope);
+        2.  **Sub-Project Maintainers** are Contributors who have earned the ability to modify (“commit”) source code, documentation, or other technical artifacts in a specific recognized **Sub-Project’s** repository(ies) (details in Section 5); and
 
-      3. organizing sub-projects and removing sub-projects;
+        3.  A Contributor may become a **Maintainer** for a particular **Sub-Project’s** repository by a simple majority approval of the existing **Maintainers** of that **Sub-Project’s** repository. Similarly, a **Maintainer** may be removed by a majority approval of the other existing **Maintainers** of that same Sub-Project’s respository.
 
-      4. creating sub-committees or working groups to focus on cross-project technical issues and requirements;
+    4.  Participation in the Project through becoming a Contributor and **Sub-Project Maintainer** is open to anyone so long as they abide by the terms of this Charter.
 
-      5. appointing representatives to work with other open source or open standards communities;
+    5.  The TSC may (1) establish workflow procedures for the submission, approval, and closure/archiving of recognized **Sub-Projects** (subject to the voting rules in Section 5), (2) set ecosystem-wide minimum requirements or guidelines for the promotion of Contributors to **Sub-Project Maintainer** status (while allowing individual **Sub-Projects** to define more specific criteria), and (3) amend, adjust, refine and/or eliminate ecosystem-level roles, and create new roles, and publicly document any TSC roles and the roles and responsibilities of **Sub-Project Maintainers**, as it sees fit.
 
-      6. establishing community norms, workflows, issuing releases, and security issue reporting policies;
+    6.  The TSC may elect a TSC Chair, who will preside over meetings of the TSC and will serve until their resignation or replacement by a TSC decision (as per Section 5). The TSC Chair, or any other TSC member so designated by the TSC, will serve as the primary communication contact between the Project and Hyperledger Foundation, a directed fund of The Linux Foundation.
 
-      7. approving and implementing policies and processes for contributing (to be published in the CONTRIBUTING file) and coordinating with the series manager of the Project (as provided for in the Series Agreement, the “Series Manager”) to resolve matters or concerns that may arise as set forth in Section 7 of this Charter;
+    7.  **Responsibilities:** The TSC will be responsible for all aspects of oversight relating to the Project, which may include:
```

Comment:
This section in general seems to mix the terms project, sub-project and repository a lot. Maybe "Project" here now means "Fabric ecosystem"? There is existing wording below talking about "sub-projects", but this (I think) relates to repositories, not "Sub-projects" that we have now defined to be different implementations or specializations of the Fabric blockchain model.